ext/session/session.c

```diff
@@ -2376,6 +2380,31 @@ static PHP_FUNCTION(session_unset)
 }
 /* }}} */
 
+/* {{{ proto int session_gc(void)
+   Perform GC and return number of deleted sessions */
+static PHP_FUNCTION(session_gc)
+{
+	zend_long num;
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	if (PS(session_status) != php_session_active) {
+		php_error_docref(NULL, E_WARNING, "Session is not active");
```

Comment:
session_gc() handler is defined in save handlers. Unless save handler storage is initialized, gc function won't work.
If you don't like stray sessions by cron tasks, you can use static session ID for the task. 

e.g.
<?php
ini_set('session.use_strict_mode', false);
session_id('ThisIsSessionIDforCronTask');
session_start();
session_gc();
?>

However, since this is C code, I can write code that open, gc, close session. In this case, session must be inactive. Otherwise, I'll destroy save handler data structure.

If above code is written. GC cron tack code would be

<?php
session_gc();
?>

Modules should do more work rather than users. I'll implement it. Is this satisfactory?